### PR TITLE
Increase header size limits

### PR DESCRIPTION
**Pull request by ChadNedzlek**
The OAuth bounce from GitHub sends some very large cookie
tokens, and the default limits are small enough that, for some
some users with many teams/claims, they get rejected with
400 : Headers too large

### CODE REVIEW

**Review by MattGal — DISMISSED** [review]

File: src/Microsoft.DotNet.ServiceFabric.ServiceHost/ServiceHostWebSite.cs [hunk 1]
```diff
@@ -28,11 +28,16 @@ public static void Run(string serviceTypeName)
...
+                    {
+                        // Default 32k, which isn't enough for oauth cookies from GitHub for people with many teams/claims
+                        o.Limits.MaxRequestHeadersTotalSize = 65536;
+                    }
+                )
                 .UseContentRoot(AppContext.BaseDirectory)
                 .ConfigureAppConfiguration((context, builder) =>
                 {
-                    builder.AddDefaultJsonConfiguration(context.HostingEnvironment, serviceProvider: null);
+                    builder.AddDefaultJsonConfiguration(context.HostingEnvironment, serviceProvider: null)
```
Comment (riarenas) [inline src/Microsoft.DotNet.ServiceFabric.ServiceHost/ServiceHostWebSite.cs:40]:
```suggestion
                    builder.AddDefaultJsonConfiguration(context.HostingEnvironment, serviceProvider: null);
```

**Review by riarenas — DISMISSED** [review]

**Review by riarenas — APPROVED** [review]

**Comment by ericstj** [comment]
Was this deployed? I'm still failing on login to maestro.
```
Bad Request - Request Too Long
HTTP Error 400. The size of the request headers is too long.
```

**Comment by MattGal** [comment]
Yes, this would have deployed on 7/20;  is there any chance you've exceeded 64 KB?

**Comment by ericstj** [comment]
I'm still hitting it `Bad Request - Request Too Long`  I have no idea if I've exceeded those thresholds.  The number of teams I belong to isn't much different than it was before.

**Pull request merged**